packages/fleet-preview/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "code": "FL",
+  "version": "1.46.97",
```

Comment:
is it correct to add the currently latest version here? or what should i put to not break anything but allow an initial package to be uploaded?

Reply:
correct. it should get updated by the main script. for now, a working version is fine.